Modules/CLI/CreateDICOMSeries/CreateDICOMSeries.xml

```diff
@@ -50,6 +50,13 @@
       <label>Study Date</label>
       <default>20060101</default>
     </string>
+    <string>
+      <name>studyTime</name>
+      <longflag>--studyTime</longflag>
+      <description><![CDATA[The time of the study [0008-0030]]]></description>
```

Comment:
Is there a good reason to introduce a new notation conventions for DICOM tags here? Why not `(0008,0030)` or `(0x0008,0x0030)`? Same for "The time of the study" - why not `StudyTime` or `Study Time`?

Reply:
I followed existing pattern. We can change all of them to standard style.

Reply:
Makes sense. If it is existing pattern, I am fine leaving them as is.